dynamic_programming/fizz_buzz.py

```diff
@@ -0,0 +1,61 @@
+# https://en.wikipedia.org/wiki/Fizz_buzz#Programming
+
+
+def fizz_buzz(number: int, iterations: int) -> str:
+    """
+    Plays FizzBuzz.
+    Prints Fizz if number is a multiple of 3.
+    Prints Buzz if its a multiple of 5.
+    Prints FizzBuzz if its a multiple of both 3 and 5 or 15.
+    Else Prints The Number Itself.
+
+    >>> fizz_buzz(1,7)
+    '1 2 Fizz 4 Buzz Fizz 7 '
```

Comment:
We need a test that generates `FizzBuzz`.

We need a test with negative numbers.

We need a test with floating point numbers.

Reply:
Okay, I'll be on it/